builtin/logical/pki/cert_util.go

```diff
@@ -895,7 +896,7 @@ func generateCreationBundle(b *backend, data *dataBundle) error {
 			notAfter.After(data.signingBundle.Certificate.NotAfter) && !data.role.AllowExpirationPastCA {
 
 			return errutil.UserError{Err: fmt.Sprintf(
-				"cannot satisfy request, as TTL is beyond the expiration of the CA certificate")}
+				"cannot satisfy request, as TTL would result in notAfter %s that is beyond the expiration of the CA certificate at %s", notAfter.Format(time.RFC3339Nano), data.signingBundle.Certificate.NotAfter.Format(time.RFC3339Nano))}
```

Comment:
Can we not have the term `notAfter` in the user response?

Reply:
This is actually an official term! https://tools.ietf.org/html/rfc5280#section-4.1.2.5

Reply:
Ah, cool!